charts/govuk-apps-conf/templates/asset-manager-nginx-configmap.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: asset-manager-nginx-conf
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
+data:
+  nginx.conf: |-
+    user nginx;
+
+    error_log /var/log/nginx/error.log warn;
+    pid /var/run/nginx.pid;
+
+    events {
+      worker_connections 1024;
+    }
+
+    http {
+      upstream asset-manager-proxy {
+        server localhost:3000;
+      }
+
+      # Set GOVUK-Request-Id if not set
+      # See http://nginx.org/en/docs/http/ngx_http_perl_module.html
+      perl_modules perl/lib;
+      perl_set $govuk_request_id '
+        sub {
+          my $r = shift;
+          my $current_header = $r->header_in("GOVUK-Request-Id");
+
+          if (defined $current_header && $current_header ne "") {
+            return $current_header;
+          } else {
+            my $pid = $r->variable("pid");
+            my $msec = $r->variable("msec");
+            my $remote_addr = $r->variable("remote_addr");
+            my $request_length = $r->variable("request_length");
+            return "$pid-$msec-$remote_addr-$request_length";
+          }
+        }
+      ';
+
+      server {
+        server_name asset-manager asset-manager.*  ;
+        # Send the Strict-Transport-Security header
+        add_header Strict-Transport-Security $sts_default;
+
+        add_header Permissions-Policy interest-cohort=();
+
+        listen 80;
+
+        proxy_set_header Host $http_host;
+
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Server $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header GOVUK-Request-Id $govuk_request_id;
+        proxy_redirect off;
+        proxy_connect_timeout 1s;
+        proxy_read_timeout 60;
+
+        access_log /var/log/nginx/asset-manager-access.log timed_combined;
+        access_log /var/log/nginx/asset-manager-json.event.access.log json_event;
+        error_log /var/log/nginx/asset-manager-error.log;
```

Comment:
As discussed, we'll need to ensure these aren't filling up ephemeral storage on the container (and are being sent to LogIt or wherever they're supposed to go). No probs if you wanna do that in a separate PR or in this one.